.github/workflows/build-and-test.yml

```diff
@@ -2,9 +2,7 @@ name: Build
 
 on:
   push:
-    branches: [ master ]
```

Comment:
Removes `branches` [filter](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#using-filters) from the `push` event trigger.

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#push

Will now trigger any time

> One or more commits are pushed to a repository branch or tag.

https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#push

Reply:
(This sounds like the desired behavior)